### PR TITLE
Fix typo on security learn page

### PR DESCRIPTION
**Pull request by krvaibhaw**
## Description

Update Ethereum security and scam prevention page

### CODE REVIEW

File: src/content/security/index.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Browser extensions like Chrome extensions or Add-ons for Firefox can augment use
 - Removing unused browser extensions
 - Install Chrome extensions locally to stop auto-updating (Advanced)
 
-[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)
+[More on the risks of browser extensions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)
```
Comment (minimalsm) [inline src/content/security/index.md:104]: Nice catch 👌

File: src/content/security/index.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ These scams usually stipulate a limited time of opportunity to claim the giveawa
 
 #### Social media hacks {#social-media-hacks}
 
-A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organizations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)
+A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organizations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoins (or $500,000 as of September 2021)
```
Comment (minimalsm) [inline src/content/security/index.md:183]: @wackerow any thoughts here?

Comment (wackerow) [reply src/content/security/index.md:183]: Traditionally have stuck with "bitcoin" for both singular and plural. 

https://github.com/bitcoinbook/bitcoinbook

Going off this respected resource as well, no use of "bitcoins" in the entire book.

Would also go with lowercase "bitcoin" for the asset (ie: above should be `...posted a bitcoin giveaway...`), and only capitalize it when referring to the protocol or network. ie. "I sent 2 bitcoin on the Bitcoin network."

Comment (krvaibhaw) [reply src/content/security/index.md:183]: Thanks for the info 😊 
Changing it to bitcoin then.

**Review by minimalsm — APPROVED** [review]
Merging, thanks again!

**Pull request merged**